src/Symfony/Component/FeatureFlag/FeatureRegistryInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\FeatureFlag;
+
+use Psr\Container\ContainerInterface;
+use Symfony\Component\FeatureFlag\Exception\FeatureNotFoundException;
+
+interface FeatureRegistryInterface extends ContainerInterface
```

Comment:
This doesn't have to extend ContainerInterface to me. It's rather an implementation detail: a core implementation could get features from a PSR-11 container (or maybe ServiceProviderInterface given `getNames()`?), another one could just have them in-memory.

Reply:
Indeed, it's not DIC-related but if we follow the spec, we can say that the `FeatureRegistry` is a Container of Features.
> The goal set by ContainerInterface is to standardize how frameworks and libraries make use of a container to obtain objects and parameters (called entries in the rest of this document).

On the other hand, in Symfony, the `ContainerInterface`  is mainly extended in the DI component. For example, `FormBuilderInterface`,  `MessageCatalogueInterface` or `ParameterBagInterface` do not extend it.

Let's removing this to keep the same logic. Is it OK for you @yceruto ?